css/.csscomb.json

```diff
@@ -13,11 +13,9 @@
     "combinator-space": [" ", " "],
     "element-case": "lower",
     "eof-newline": true,
-    "leading-zero": null,
```

Comment:
Fair point. To be thorough, is this rule also disabled in our sass-lint ruleset?